tools/publishIsolate.ts

```diff
@@ -37,40 +36,24 @@ async function main() {
   for (const packageName of publishPackageList) {
     console.log(`Publishing ${packageName}`);
     const packagePath = path.join(isolatePackageRoot, packageName);
-    const command = `npm publish --token ${npmAuthToken} ${packagePath}`;
+    const command = `yarn publish ${packagePath}`;
```

Comment:
Can this command work?

I didn't find any documentation about `yarn publish <path>`.

Reply:
https://classic.yarnpkg.com/lang/en/docs/cli/publish/

see `yarn publish [folder]`

Reply:
That's good.